.eslintrc.js

```diff
@@ -56,7 +56,6 @@ module.exports = {
         props: false,
       },
     ],
-    ignoreDestructuring: 'error',
```

Comment:
Why is it deleted here?

Reply:
because this rule will cause strange errors and i don't know how to fix it